civihr_employee_portal/civihr_employee_portal.module

```diff
@@ -85,11 +85,12 @@ function civihr_employee_portal_init() {
 
     // Add our additional css libraries
     drupal_add_css(drupal_get_path('module', 'civihr_employee_portal') . "/css/custom.css");
+    drupal_add_css("sites/all/modules/civicrm/tools/extensions/civihr/org.civicrm.bootstrapcivicrm/css/bootstrap-civicrm-style.css");
```

Comment:
This could be rewritten as
```
drupal_add_css(drupal_get_path('module', 'civicrm') . "/tools/extensions/civihr/org.civicrm.bootstrapcivicrm/css/bootstrap-civicrm-style.css");
```
I am afraid that this is too specific, I think it can happen that the extensions will not be placed under `tools/extensions` (i believe one of our pilot projects has indeed this kind of setup), and as such this line will end up including a non-existing file

Can you check with the other guys (or even the core guys), if it's possible at all to have access to the extensions path in drupal? Or if there any other solution that could work in any scenario